src/clic.adoc

```diff
@@ -898,36 +695,48 @@ spintstatus
     28  spp           Previous privilege mode, same as sstatus.spp
     27  spie          Previous interrupt enable, same as sstatus.spie
  26:24  (reserved)
- 23:16  spil[7:0]     Previous interrupt level
+ 23:16  spipprio[7:0] Previous interrupt preemption priority
  15:12  (reserved)
  11:0   exccode[11:0] Exception/interrupt code, same as scause[11:0]
 ----
 
 The supervisor {scause} register has only a single `spp` bit (to
 indicate user/supervisor) mirrored from {sstatus}.`spp`.
 
-==== New Interrupt-Level Threshold ({sintthresh}) CSR
-
-The interrupt-level threshold ({sintthresh}) is a new read-write WARL CSR,
-which holds an 8-bit field (`th`) for the threshold level of the
-supervisor privilege mode.  The `th` field is held in the least-significant
-8 bits of the CSR, and zero should be written to the upper bits.
-
-=== ssclic CLIC Reset Behavior
+=== Reset Behavior
 
-Interrupt level 0 corresponds to regular
+Interrupt preemption priority IPPRIO_MAX corresponds to regular
 execution outside of an interrupt handler.
 
 NOTE: For an S-mode execution environment, the EEI should specify
 that {sstatus}.`sie` is also reset on entry. It is then responsibility of
 the execution environment to ensure that is true before beginning execution
 in S-mode.
 
+=== Interrupt Operation
+
+This section describes the operation of ssclic interrupts.
+The behavior is modified in analogy to smclic.
+
+==== Trap behavior
+
+In CLIC mode, when a trap _i_ is taken,
+the current value of {sintstatus}.{sipprio} is written to {spintstatus}.{spipprio}.
+If the trap is taken on an interrupt, {sintstatus}.{sipprio} is updated to `iprio[__i__]`.
+
+NOTE: Synchronous exceptions do not modify the value of {sintstatus}.{sipprio}.
+
+==== Returns from Handlers
+
+The behavior of an {sret} instruction is modified as follows:
+
+In CLIC mode, {sret} sets {sintstatus}.{sipprio} to {spintstatus}.{spipprio}.
+
 ==== State Enable
 
 If the Smstateen extension is implemented, then the bit 53 (CLIC) in mstateen0 is
 implemented. If bit 53 (CLIC) of a controlling mstateen0 CSR is zero, then
-access to the new CSRs ({sivt}, {sintthresh})
+access to the new CSRs ({sivt})

```

Comment:
```suggestion
access to the new CSR ({sivt})
```
Changed from plural to singular since there's only one CSR left in the list

Reply:
fixed